autoload/neomake.vim

```diff
@@ -247,11 +249,20 @@ function! s:AddExprCallback(maker) abort
             call Func(entry)
         end
 
-        if entry.valid && !file_mode
+        if !entry.valid
+            if get(a:maker, "remove_invalid_entries", 0)
```

Comment:
Also, you shouldn't need to use `get` here. Since you added that property to the list of defaults, that property will always be available on the maker.